cobalt/media/base/sbplayer_pipeline.cc

```diff
@@ -1049,6 +1049,8 @@ void SbPlayerPipeline::OnDemuxerStreamRead(
     for (const auto& buffer : buffers) {
       playback_statistics_.OnAudioAU(buffer);
       if (!buffer->end_of_stream()) {
+        last_audio_sample_interval_ =
+            buffer->timestamp() - timestamp_of_last_written_audio_;
```

Comment:
Consider using std::max(..., 0) here to ensure that the result isn't negative.  It shouldn't really happen in production, but this is technically possible.